cosmoz-image-viewer.html.js

```diff
@@ -142,9 +105,15 @@ export const template = html`
 		opacity: 0.8;
 		font-size: 0.8em;
 	}
+
+	cosmoz-slider { 
+		min-height: 150px;
+		overflow-y: auto !important;
+		height:100%;
+	}
 </style>
 
-<div>
+<div style="height: 100%">
```

Comment:
Do we need this wrapper or can we remove it? :host is enough wrapper ?

Reply:
I'll try to remove it.

Reply:
@megheaiulian I've removed it and it works fine.